packages/api-client/src/components/ApiClient/Request/RequestAuthentication/SecurityScheme.vue

```diff
@@ -368,6 +486,83 @@ const startAuthentication = (url: string) => {
           </CardFormGroup>
         </CardFormRows>
       </template>
+      <!-- Client Credentials Flow -->
+      <template
+        v-else-if="
+          (value as OpenAPIV3.OAuth2SecurityScheme).flows.clientCredentials
+        ">
+        <template v-if="authentication.oAuth2.accessToken">
```

Comment:
Essentially blanket copy / pasted this from the implicit flow to give feedback on the fact that a token was successfully fetched / allow for reseting the token.

This currently looks like the following:
![image](https://github.com/scalar/scalar/assets/8587567/c904f8a6-b26f-48fb-bc22-66ad6c37dfb8)

Can split this into a separate row / remove entirely for now / do something else if this looks a little too wonky.